.bitmap

```diff
@@ -1266,6 +1266,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/lanes/ui/drawer"
     },
+    "ui/dropdown": {
```

Comment:
I am working on a UI component as well with Amir called : teambit.dot-lanes/inputs/lane-selector.  would love to compose one you create as well. 
I believe this component should be called: `teambit.lanes/navigation/lane-switcher` and it should use a UI component called: `teambit.dot-lanes/inputs/lane-selector` which i would also use from dot-lanes.

Can you use the design created with @erezbit and amir? you will need to align with @guysaar223 on a mutual interface for LaneModel and LaneDescriptor. (very important) if you plan on using it so i could reuse this as well

Reply:
makes sense, fixed

Reply:
Looked at the LaneDescriptor; It can be a drop in replacement for wherever LaneModel is being used. I will touch base with @guysaar223  to align